src/components/Login/useOpenGoogleSignIn.ts

```diff
@@ -0,0 +1,7 @@
+export const useOpenGoogleSignIn = () => {
+  return (isHidden: boolean): void => {
+    const getElem = document.getElementById('signInDiv');
+    if (!getElem) return;
+    getElem.hidden = isHidden;
+  };
+};
```

Comment:
Hmm, it would be better to pass the element id in through as the method's parameter instead of using a fixed value here, since it might change in the actual div and you might forget to update it here